config/socket/game.js

```diff
@@ -323,9 +353,9 @@ class Game {
     });
   }
   /**
-   *
-   * @returns {void} description
-   * @param {any} cb
+   * @description this method triggers get all answer controller
+   * @returns {void}
+   * @param {function} cb
```

Comment:
what does Cb mean?. Is there a better way to name this parameter?

Reply:
Well Cb is in the codebase before but i will make changes to it.